demo.html

```diff
@@ -0,0 +1,414 @@
+<!DOCTYPE html>
+<html lang = "en">
+<head>
+  <meta charset="UTF-8">
+  <title>Build Your Own Bootstrap</title>
+  <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300.700|Raleway" rel="stylesheet">
+</head>
+<body>
+    <div>
+      <header id="main-header" class="flex-container">
+        <p id="title">Vast CSS</p>
+        <p id="sub-title">'Cause Bootstrap is B.S.</p>
+      </header>
+    </div>
+
+    <div class="body-content">
+
+    <header class="sections" id="headings-section">
+      <h1 class="section-title">Headings</h1>
+
+      <div class="breakdown">
+      <p>To style headings in various sizes, add the
+        <code class="class-name">.vast-h1</code> through
```

Comment:
nice usage of the code html tag --- ive never used it and it looks appropriate here!